DSCResources/xTimeZone/xTimeZone.psm1

```diff
@@ -13,7 +13,7 @@ function Get-TargetResource
     )
 
     #Get the current TimeZone
-    $CurrentTimeZone = Invoke-Expression "tzutil.exe /g"

```

Comment:
I can't see any difference in the compare, the version in my AddTests branch does not include any trailing white space.  I may have removed white space if it was there. Other than that the lines are the same.